src/resources/LinksProducts.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ *  @module Resources/Products
+ */
+
+import { AuthParams, SendData } from "../api/RestAPI.js";
+import { CRUDItemsResponse, CRUDResource } from "./CRUDResource.js";
+import { DefinedRoute } from "./Resource.js";
+import { TransactionTokenType } from "./TransactionTokens.js";
+
+export type LinkProductListItem = {
```

Comment:
Also I did not include the active parameter in the public ones. I may want to remove the platformId as well, right?

Reply:
Yeah `platformId` and `active` are omitted for non admins: https://github.com/univapaycast/services-typescript/blob/ea988d2a1f516fad8ebb496439edb550dc39ee47/services/checkout/src/handlers/links_products/list.ts#L34